Lpc compute docs (#2651)
buildkite/docs: pages/buildkite_compute/source_control.md

> ## Compute code access

Reviewer: ```suggestion
## Hosted agents code access
```

> This page details changes that need to be made to the GitHub integration to provide code access to the Buildkite compute agents see [Pipeline Migration](/docs/buildkite-compute/pipeline-migration) for changes that may need to be completed at an individual pipeline level.

Reviewer: ```suggestion
This page details changes that need to be made to the GitHub integration to provide code access to the Buildkite hosted agents.

To learn more about changes that may need to be completed at an individual pipeline level, see [Pipeline migration](/docs/buildkite-compute/pipeline-migration).
```

### Private repositories

> During the trial we only support accessing private repositories through GitHub, if you have requirements to access private repositories from another source control service please contact support.

Reviewer: ```suggestion
During the trial, access to private repositories is only support through GitHub. If you have requirements to access private repositories from another source control service, please contact support.
```

> To use a private GitHub repository with Buildkite compute services you will need to authorize Buildkite to access your repository.

Reviewer: ```suggestion
To use a private GitHub repository with Buildkite hosted agent services you will need to authorize Buildkite to access your repository.
```

> 1. Navigate to your Buildkite org settings page [here](https://buildkite.com/organizations/~/settings).

Reviewer: Removing the unnecessary 'here'.

```suggestion
1. Navigate to your [Buildkite organization's settings page](https://buildkite.com/organizations/~/settings).
```
